index.bs

```diff
@@ -749,6 +759,14 @@ agent MUST run the following steps:
    congestion control algorithms that optimize for |congestionControl|, as allowed by
    [[!RFC9002]] [Section 7](https://www.rfc-editor.org/rfc/rfc9002#section-7),
    then set |congestionControl| to `"default"`.
+1. Let |protocols| be {{WebTransport/constructor(url, options)/options}}'s
+   {{WebTransportOptions/protocols}}
+1. If any of the values in |protocols| occur more than once, fail to match
+   the requirements for elements that comprise the value of `WebTransport-Subprotocol`
```

Comment:
Can we drop the "sub" and use `WebTransport-Protocol` instead?

Reply:
We seem to be going the other way naming-wise judging from other feedback I've received.

But here specifically it needs to match https://github.com/ietf-wg-webtrans/draft-ietf-webtrans-http3/pull/144 which is already merged.